engine/install/rhel.md

```diff
@@ -0,0 +1,271 @@
+
+$ sudo yum-config-manager \
+    --add-repo \
+    {{ download-url-base }}/docker-ce.repo
+```
+
+> **Optional**: Enable the **nightly** or **test** repositories.
+>
+> These repositories are included in the `docker.repo` file above but are disabled
+> by default. You can enable them alongside the stable repository.  The following
+> command enables the **nightly** repository.
+>
+> ```console
+> $ sudo yum-config-manager --enable docker-ce-nightly
+> ```
+>
+> To enable the **test** channel, run the following command:
+>
+> ```console
+> $ sudo yum-config-manager --enable docker-ce-test
+> ```
+>
+> You can disable the **nightly** or **test** repository by running the
+> `yum-config-manager` command with the `--disable` flag. To re-enable it, use
+> the `--enable` flag. The following command disables the **nightly** repository.
+>
+> ```console
+> $ sudo yum-config-manager --disable docker-ce-nightly
+> ```
+>
+> [Learn about **nightly** and **test** channels](index.md).
+
+#### Install Docker Engine
+
+1.  Install the _latest version_ of Docker Engine and containerd, or go to the next step to install a specific version:
+
+    ```console
+    $ sudo yum install docker-ce docker-ce-cli containerd.io
+    ```
+
+    If prompted to accept the GPG key, verify that the fingerprint matches
+    `060A 61C5 1B55 8A7F 742B 77AA C52F EB6B 621E 9F35`, and if so, accept it.
+
+    > Got multiple Docker repositories?
+    >
+    > If you have multiple Docker repositories enabled, installing
+    > or updating without specifying a version in the `yum install` or
+    > `yum update` command always installs the highest possible version,
+    > which may not be appropriate for your stability needs.
+
+    Docker is installed but not started. The `docker` group is created, but no users are added to the group.
+
+2.  To install a _specific version_ of Docker Engine, list the available versions
+    in the repo, then select and install:
+
+    a. List and sort the versions available in your repo. This example sorts
+       results by version number, highest to lowest, and is truncated:
+
+    ```console
+    $ yum list docker-ce --showduplicates | sort -r
+
+    docker-ce.s390x                3:20.10.7-3.el8                 docker-ce-stable
+    ```
+
+    The list returned depends on which repositories are enabled, and is specific
+    to your version of RHEL (indicated by the `.el8` suffix in this example).
+
+    b. Install a specific version by its fully qualified package name, which is
+       the package name (`docker-ce`) plus the version string (2nd column)
+       starting at the first colon (`:`), up to the first hyphen, separated by
+       a hyphen (`-`). For example, `docker-ce-20.10.7`.
+
+    ```console
+    $ sudo yum install docker-ce-<VERSION_STRING> docker-ce-cli-<VERSION_STRING> containerd.io
+    ```
+
+    Docker is installed but not started. The `docker` group is created, but no users are added to the group.
```

Comment:
This command installs Docker, but it doesn't start Docker. It also creates a `docker` group, however, it doesn't add any users to the group by default.